resources/metadata/gctsCreateRepository.yaml

```diff
@@ -80,3 +80,5 @@ spec:
         - STAGES
         - STEPS
         default: GIT
+        possibleValues:
+        - GIT
```

Comment:
@gochax are there any other values accepted?

Reply:
currently this is the only possible value. this will change in the future.